cromwell-logging/src/main/scala/cromwell/pubsub/PubSubMediator.scala

```diff
@@ -0,0 +1,63 @@
+package cromwell.pubsub
+
+import akka.actor.{Props, Actor, ActorRef, ActorSystem}
+
+/**
+  * This is a wrapper to Eventstream as one of the event bus with SubChannel trait,
+  * it is defined inside business logging component but later it needs to moved out
+  * to separate place.
+  */
+//TODO:Needs to move some other place
+trait PubSubMediator{
+  /**
+    *Subscribes consumer function(topic,message) where topic is event structure and message is event data.
+    * @param actorRef actor that needs to be subscribed,this Actor class is of type (T,Any) where
+    *                 T is generic topic and Any is any payload
+    * @param system system actor system for creating subscriber actor this is implicit
+    * @tparam T
+    * @return
+    */
+  def subscribe[T](actorRef: ActorRef)(implicit system:ActorSystem):Unit = {
+    system.eventStream.subscribe(actorRef , classOf[(T, Any)])
```

Comment:
Well i am kinda want to separate event topic from event payload, Above example club them into one. Because you probably need different types of  topic like WorkflowEvent, MonitoringEvent, BackendEvent, EgineEvent and hence each has different payload.